doc/api/path.md

```diff
@@ -261,12 +261,18 @@ path.isAbsolute('.')        // false
 On Windows:
 
 ```js
-path.isAbsolute('//server')  // true
-path.isAbsolute('C:/foo/..') // true
-path.isAbsolute('bar\\baz')  // false
-path.isAbsolute('.')         // false
+path.isAbsolute('//server')    // true
+path.isAbsolute('\\\\server')  // true
+path.isAbsolute('C:/foo/..')   // true
+path.isAbsolute('C:\\foo\\..') // true
+path.isAbsolute('bar\\baz')    // false
+path.isAbsolute('bar/baz')     // false
+path.isAbsolute('.')           // false
 ```
 
+*Note*: On Windows, the `path.isAbsolute()` method will accept both forward
+slash (`/`) and backwards slash (`\`) characters as path delimiters.
```

Comment:
btw, this sounds a bit overly specific, although I might be wrong… is there anything in the Node API where `/` isn’t accepted on Windows?